ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -544,6 +702,9 @@ private[testtool] abstract class CommandDeduplicationBase(
     case _ =>
       throw new IllegalArgumentException(s"Invalid timeout scale factor: $timeoutScaleFactor")
   }
+
+  private def absoluteLedgerOffset(value: String) =
```

Comment:
Nit: This can go to the companion, right? Maybe sth else too.

Reply:
I think many utilities can be moved to the companion but it has been used so far only for types. I am unopinionated about this.